common/workunit/workunit.cpp

```diff
@@ -13650,9 +13661,13 @@ void executeThorGraph(const char * graphName, IConstWorkUnit &workunit, const IP
                 if (it == expectedStates.end())
                     throw makeStringExceptionV(0, "Query %s failed, state: %s", wuid.str(), getWorkunitStateStr(state));
             }
+            if (!blockedTime)
```

Comment:
If it reaches here, blockedTime cannot already be set here, so no point in this test.
Either it was wait state and broke out, or it running (1st transition/1st time through loop), or an unexpected state.